### PR TITLE
Session 1, 2024

### LINKED ISSUE #8: Prep for 'foundations of ggplot2'
- [x] Update slides from last year
- [x] Update notes and other materials if needed
- [x] Update this book to link to or embed those things

### PR DESCRIPTION
**Pull request by Aariq**
Adding materials for session 1 (closes #8)

Still to-do:
- [x] Add final slide with links to drop-in hours, incubator, etc.
- [x] Cut out some examples especially one of the two possible worked examples relating to "stats"
- [x] Make a notes.qmd with the code from all the worked examples
- [x] Link that notes.qmd in the index.qmd

### CODE REVIEW

File: 2024/01-foundations/slides.qmd [hunk 1]
```diff
@@ -1,8 +1,764 @@
...
+#| output-location: fragment
+#| code-line-numbers: "2-3|"
+ggplot(penguins, aes(x = island, y = body_mass_g, color = sex)) +
+  geom_boxplot(color = "black") +
+  geom_jitter(color = "darkred")
+```
+
+## Scales
+
+-   `scale_` functions effect the range (limits) and breaks of scales and the labels and appearance of corresponding guides.
```
Comment (diazrenata) [inline 2024/01-foundations/slides.qmd]: affect

**Review by diazrenata — COMMENTED** [review]
This looks awesome to me! I made one note on a typo. Does it feel like a manageable amount for the time?

**Comment by Aariq** [comment]
> This looks awesome to me! I made one note on a typo. Does it feel like a manageable amount for the time?

I'm going to test it out for time right now and see.  There are two alternative worked examples for the "statistics" section—one for `stat_summary()` and one for using `after_stat()` with `geom_histogram()`.  I don't think I will have time to do both.  Do you think one is more appropriate than the other?

**Review by KristinaRiemer — APPROVED** [review]
I think this looks great! I'm excited about tomorrow.

**Comment by Aariq** [comment]
Going to go ahead and merge this ahead of the workshop.  Will make a new PR if there are last-minute changes

**Pull request merged**